onmt/decoders/transformer.py

```diff
@@ -12,21 +12,41 @@
 
 
 class TransformerDecoderLayer(nn.Module):
-    """
+    """Transformer Decoder layer block in Pre-Norm style.
+
+    .. mermaid::
```

Comment:
I reckon this graph is a simplified representation of pre-norm. Maybe add a title or a sentence to explain?